File: neraquasar-domain/src/main/java/org/neraquasar/domain/Account.java
package org.neraquasar.domain;

import java.util.Set;

/**
 * Учётная запись.
 *
 * @author Konstantin Valer'evich Dichenko
 *         Created 01.05.2017
 */
public class Account {

    private Person person;

    private String username;

    private String password;

    private Set<Role> roles;
}

File: neraquasar-domain/src/main/java/org/neraquasar/domain/Account.kt
package org.neraquasar.domain

/**
 * Учётная запись.
 *
 * @author Konstantin Valer'evich Dichenko
 * Created 01.05.2017
 */
data class Account(val person: Person,
                   var username: String,
                   var password: String = "",
                   val roles: Set<Role> = HashSet<Role>())
